### ISSUE TITLE
PiAppsTerminalAdvanced error with wine install 

### ISSUE THREAD

**Issue by PUP-Loki** [Botspot/pi-apps#632]
When trying to install wine i receive this error
pi-apps install 'Wine (x86)'
The Wine (x86) App Exists
sh: 1: Syntax error: "(" unexpected

**Comment by Botspot** [comment]
@Itai-Nelken

**Comment by Itai-Nelken** [comment]
@techcoder20 made the app, mine isn't in pi-apps

**Comment by techcoder20** [comment]
I am going to say its the script problem as its a sh error. PiAppsTerminalAdvanced is written entirely in python so it should not give an sh error if its a problem in the script. The only bash command that runs after the wine app is recognized is the pi-apps manage command: `os.system("~/pi-apps/manage multi-install " + AppNameNoSpace)` so i suggest checking out the wine script once.

**Comment by CleanMachine1** [comment]
That line looks wrong to me, it is executing
 "~/pi-apps/manage multi-install " + AppNameNoSpace
in bash

I don't think you can have python operators in an os.system.

If not prove me wrong, at least I'd be learning it

**Comment by PUP-Loki** [comment]
been playing round if i remove ( ) from the x86 then it installs without an issue

**Comment by Botspot** [comment]
> been playing round if i remove ( ) from the x86 then it installs without an issue

Are you saying this works??
```
pi-apps install 'Wine x86'
```
It would seem to me that that app name does not exist. So how did you get this working?

**Comment by Itai-Nelken** [comment]
I think the issue is that the command that bash is getting is this: `~/pi-apps/manage multi-install Wine (x86)` instead of `~/pi-apps/manage multi-install "Wine (x86)"` for example.
@techcoder20 I think the command should be
```py
os.system("~/pi-apps/manage multi-install " + "\"" + AppNameNoSpace + "\"")
```
(not tested)

**Comment by Botspot** [comment]
> Oh yes that might be the problem. I will fix it :)

@techcoder20, what's the current status of this issue? Can it be closed?

**Comment by techcoder20** [comment]
@Botspot It has been fixed https://github.com/techcoder20/PiAppsTerminalAdvanced/commit/ca716557855dd3086187b9b750ac24c17b0419f9

**Comment by Botspot** [comment]
> @Botspot It has been fixed [techcoder20/PiAppsTerminalAdvanced@ca71655](https://github.com/techcoder20/PiAppsTerminalAdvanced/commit/ca716557855dd3086187b9b750ac24c17b0419f9)
